container-images/mds-micromobility-policy-author/webpack.config.ts

```diff
@@ -0,0 +1,3 @@
+import webpack from '@mds-core/mds-webpack-config'
+
+export default webpack.Bundle('../../packages/mds-micromobility-policy-author/server').UsingDefaultConfig()
```

Comment:
Do you think we need to have a new package for this in `mds-core`? I think that we could just keep the policy & policy-author `server.ts` files around, and just inject the proper types/validators from there, then invoke the relevant `server.ts` files from `container-images/mds-policy{-author}` as we did previously. The policy & policy-author packages should be able to export other stuff, even if their `server.ts` files are vanilla MDS flavored (without any additional props taken into account).

Reply:
I do feel like `mds-core` is getting cluttered, but I feel that adding a new package makes it clearer that `mds-policy-author` is the base policy author package. I think that simply leaving `server.ts` with the micromobility type injected into it could be misleading.